gst/tensor_filter/tensor_filter.c

```diff
@@ -530,6 +540,42 @@ gst_tensor_filter_set_property (GObject * object, guint prop_id,
         g_strfreev (str_types);
       }
       break;
+    case PROP_INPUTNAME:
+      g_assert (!prop->input_configured && value);
+      /* Once configures, it cannot be changed in runtime */
+      {
+        int i;
+        gchar **str_names;
+
+        str_names = g_strsplit (g_value_get_string (value), ",", -1);
+        prop->input_meta.num_tensors = g_strv_length (str_names);
+
+        for (i = 0; i < prop->input_meta.num_tensors; i++) {
+          prop->input_meta.info[i].name = str_names[i];
```

Comment:
g_strfreev() frees contained strings, so have to use g_strdup.

Reply:
it's applied